### PR TITLE
*: make the help message standard

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
The format of our help message is not unified.

### What is changed and how it works?
After referring to some Linux commands like `cat`, `ls`, etc. , they use lower case and without periods in the help message. This PR makes changes to all binaries in PD repo.

### Check List 

Tests 

 - No code

### Release note 
- No release note

### CODE REVIEW

File: tools/pd-tso-bench/main.go [hunk 1]
```diff
@@ -33,9 +33,9 @@ var (
 	pdAddrs     = flag.String("pd", "127.0.0.1:2379", "pd address")
 	concurrency = flag.Int("C", 1000, "concurrency")
```
Comment (lhy1024) [inline tools/pd-tso-bench/main.go:34]: how about lower case

Comment (rleungx) [reply tools/pd-tso-bench/main.go:34]: This PR will only modify the help message. We can raise another PR to do it if it's necessary.

**Pull request merged**